java/src/jmri/jmrix/loconet/spjfile/SpjFile.java

```diff
@@ -449,7 +449,7 @@ public String getName() {
 
         public void setName(String name) {
             if (name.length() > 72) {
```

Comment:
Just a quick question, probably for @danielb987:  Why do we care that filenames are 72 characters or less long?  Does this include the path? (I guess that's two questions)

Reply:
You added the code 15 years ago in https://github.com/JMRI/JMRI/commit/46198c1975f8e9c8fb8df2c0f4252af9a178eb5b. See line 348 in that commit.